book/src/docker.md

```diff
@@ -3,17 +3,67 @@
 This repository has a `Dockerfile` in the root which builds an image with the
 `lighthouse` binary installed.
 
-To use the image, first build it (this will likely take several minutes):
+A pre-built image is available on Docker Hub and the
+[sigp/lighthouse](https://github.com/sigp/lighthouse-docker) repository
+contains a full-featured `docker-compose` environment.
+
+## Obtaining the Docker image
```

Comment:
I would add a comment here specifying that you can either use Docker Hub OR build the docker image from source.
e.g. "There are two ways of obtaining the docker image, either via Docker Hub or build the Docker Image from source. Once you have the docker image, proceed to [using the docker image](https://lighthouse-book.sigmaprime.io/docker.html#Building-the-Docker-Image)" ?